run.go

```diff
@@ -100,7 +101,7 @@ func run(context *cli.Context) error {
 
 	detach := context.Bool("detach")
 
-	if !detach {
+	if !detach && isatty.IsTerminal(os.Stdout.Fd()) {
```

Comment:
This is basically:
```Go
func IsTerminal(fd uintptr) bool {
	var termios syscall.Termios
	_, _, err := syscall.Syscall6(syscall.SYS_IOCTL, fd, ioctlReadTermios, uintptr(unsafe.Pointer(&termios)), 0, 0, 0)
	return err == 0
}
```
Could we just copy this implementation rather than pulling yet another dependency ?
["A little copying is better than a little dependency."](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s)